src/source/Sctp/Sctp.c

```diff
@@ -305,14 +305,17 @@ STATUS handleDcepPacket(PSctpSession pSctpSession, UINT32 streamId, PBYTE data,
     ENTERS();
     STATUS retStatus = STATUS_SUCCESS;
     UINT16 labelLength = 0;
+    UINT16 protocolLength = 0;
 
     // Assert that is DCEP of type DataChannelOpen
     CHK(length > SCTP_DCEP_HEADER_LENGTH && data[0] == DCEP_DATA_CHANNEL_OPEN, STATUS_SUCCESS);
 
-    MEMCPY(&labelLength, data + 8, SIZEOF(UINT16));
-    putInt16((PINT16) &labelLength, labelLength);
+	MEMCPY(&labelLength, data + 8, SIZEOF(UINT16));
```

Comment:
Nice! Would have loved this file to have more meaningful comments referencing the specs